frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/SelectedNodeInputOutputTab.tsx

```diff
@@ -88,12 +101,26 @@ const SelectedNodeInputOutputTab: React.FC<SelectedNodeInputOutputTabProps> = ({
     [getExecutionValueFromInputType],
   );
 
-  if (!task.inputs && !task.outputs) {
+  if (!task.inputs && !task.outputs && !execution) {
```

Comment:
we should not do this becuase if a task has not started yet it will not show anything where there is some data that could be shown.